v2/azure-arm.yaml

```diff
@@ -837,12 +864,14 @@ objectModelConfiguration:
         $azureGeneratedSecrets:
           - AdminCredentials
           - UserCredentials
+        $importable: false
```

Comment:
minor: Might be helpful to always put this after the `$supportedFrom`? My eyes at least have a harder time parsing these changes when it's after a (possibly long-ish) list of other things.

Reply:
Good suggestion - but I'll leave things the way they are, given I'll shortly be removing all of these.